### PR TITLE
added new call ServiceForceRefresh

### PR DESCRIPTION
**Pull request by schubi2**
This is a try. But perhaps Lada has another/better solution for.

### CODE REVIEW

File: src/PkgFunctions.h [hunk 1]
```diff
@@ -791,6 +792,8 @@ class PkgFunctions
 	/* TYPEINFO: boolean(string)*/
 	YCPValue ServiceRefresh(const YCPString&);
 	/* TYPEINFO: string(string)*/
```
Comment (mvidner) [inline src/PkgFunctions.h]: No, `TYPEINFO: boolean(string)` like for `ServiceRefresh`.

File: src/Service.cc [hunk 2]
```diff
@@ -359,12 +359,12 @@ YCPValue PkgFunctions::ServiceSet(const YCPString &old_alias, const YCPMap &serv
 }
 
 /**
-   @builtin ServiceRefresh
-   @short Refresh the service, the service must already be saved on the system!
+   @builtin ServiceRefreshHelper
+   @short Helper call for refreshing services
    @param alias alias of the service to refresh
```
Comment (mvidner) [inline src/Service.cc]: > @param force force refresh even if TTL is not reached

**Comment by mvidner** [comment]
LGTM. I think the approach is OK.

**Comment by schubi2** [comment]
OK, thanks all !!!

**Pull request merged**